main.go

```diff
@@ -52,8 +54,11 @@ func main() {
 	flag.Int("pprofPort", 0, "The port to use for the pprof http endpoints")
 	flag.String("web3", "https://mainnet.optimism.io", "Web3 RPC Optimism endpoint")
 
-	//flag.String("neynarSignerUUID", "", "neynar signer UUID")
+	// bot flags
+	flag.Uint64("botFid", 0, "FID to be used for the bot")
 	flag.String("neynarAPIKey", "", "neynar API key")
+	flag.String("neynarSignerUUID", "", "neynar signer UUID")
+	flag.String("neynarWebhookSecret", "cUIJfBD1Jn9oQC3hy_jRL9Dox", "neynar Webhook shared secret")
```

Comment:
Can this webhook secrtet be public information?

Reply:
ups nop :P but it is already revoked